instrumentation/logback/logback-access-jetty-12.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/logback/access/jetty/v12_0/AccessEventMapper.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.logback.access.jetty.v12_0;
+
+import ch.qos.logback.access.common.spi.IAccessEvent;
+import io.opentelemetry.api.logs.LogRecordBuilder;
+import io.opentelemetry.api.logs.LoggerProvider;
+import io.opentelemetry.api.logs.Severity;
+import io.opentelemetry.semconv.ClientAttributes;
+import io.opentelemetry.semconv.HttpAttributes;
+import io.opentelemetry.semconv.ServerAttributes;
+import io.opentelemetry.semconv.UrlAttributes;
+import io.opentelemetry.semconv.UserAgentAttributes;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+public class AccessEventMapper {
+
+  public static final String ACCESS_EVENT_NAME = "access-event";
+
+  private static void mapLoggingEvent(LogRecordBuilder builder, IAccessEvent accessEvent) {
+    URI uri = null;
+    try {
+      uri = new URI(accessEvent.getRequest().getRequestURL().toString());
+    } catch (URISyntaxException ignored) {
+      // ignore
+      // for parsing URI, the input itself is actually one URI, so there is low chance the
+      // exception would be thrown. Even when it actually happened, there is not much we can do
+      // here.
+    }
+
+    builder
```

Comment:
I'd imagine that setting the context with `setContext(Context.current())` could also be useful here to allow for correlating traces and logs. Idk whether the context is available here.